src/content/docs/new-relic-solutions/observability-maturity/operational-efficiency/data-governance-optimize-ingest-guide.mdx

```diff
@@ -627,6 +627,101 @@ It's not surprising that a complex and decentralized system like Kubernetes has
 
+### Filtering Namespaces
+
+The Kubernetes Integration v3 and above allows filtering which namespaces are scraped by labelling them. By default all namespaces are scraped.
+
+We use the `namespaceSelector` in the same way Kubernetes does. In order to include only namespaces matching a label, change the `namespaceSelector` by adding the following to your `values-newrelic.yaml`, under the `newrelic-infrastructure` section:
+
+```yaml
+common:
+  config:
+    namespaceSelector:
+      matchLabels:
+        key1 : "value1"
+```
+
+In this example only namespaces with the label `newrelic.com/scrape` set to `true` will be scraped:
+
+```yaml
+global:
+  licenseKey: _YOUR_NEW_RELIC_LICENSE_KEY_
+  cluster: _K8S_CLUSTER_NAME_
+
+# ... Other settings as shown above
+
+# Configuration for newrelic-infrastructure
+newrelic-infrastructure:
+  # ... Other settings as shown above
+  common:
+    config:
+      namespaceSelector:
+        matchLabels:
+          newrelic.com/scrape: "true"
+```
+
+You can also use Kubernetes expressions to include or exclude namespaces using the following syntax:
+
+```yaml
+common:
+  config:
+    namespaceSelector:
+      matchExpressions:
+      - {key: newrelic.com/scrape, operator: NotIn, values: ["false"]}
+      - {key: key1, operator: In, values: ["value1"]}
+```
+
+<Callout variant="tip">
+  The expressions under `matchExpressions` are concatenated.
+</Callout>
+
+In this example namespaces with the label `newrelic.com/scrape` set to `false` will be excluded:
+
+```yaml
+global:
+  licenseKey: _YOUR_NEW_RELIC_LICENSE_KEY_
+  cluster: _K8S_CLUSTER_NAME_
+
+# ... Other settings as shown above
+
+# Configuration for newrelic-infrastructure
+newrelic-infrastructure:
+  # ... Other settings as shown above
+  common:
+    config:
+      namespaceSelector:
+        matchExpressions:
+        - {key: newrelic.com/scrape, operator: NotIn, values: ["false"]}
+```
+
+See a full list of the settings that can be modified in the [chart's README file](https://github.com/newrelic/nri-kubernetes/tree/main/charts/newrelic-infrastructure).
+
+#### How can I know which namespaces are excluded? [#excluded-namespaces]
+
+All the namespaces within the cluster are listed thanks to the `K8sNamespace` sample. The `nrFiltered` attribute determines whether the data related to the namespace is going to be scraped.
+
+Use this query to know which namespaces are being monitored:
+
+```sql
+FROM K8sNamespaceSample SELECT displayName, nrFiltered WHERE clusterName = <clusterName> SINCE 2 MINUTES AGO
```

Comment:
@JimHagan how about prettifying this query?

SELECT displayName, nrFiltered 
FROM K8sNamespaceSample 
WHERE clusterName = INSERT_NAME_OF_CLUSTER 
SINCE 2 MINUTES AGO